inst/shinys/Rescale/app.R

```diff
@@ -20,42 +20,47 @@ ui <- fluidPage(
 
 server <- function(input,output){
   output$best_fit <- renderText({
-    
+
     set.seed(19)
     x <- rnorm(50, 2, 10)
     y <- 10 + 3*x + rnorm(50, 0, 10)
-    
+
     s_x <- input$scale_x
     s_y <- input$scale_y
-    
-    best_slope <- (cov(x, y)/var(x))
-    best_inter <- mean(y) - best_slope*mean(x)
-    
-    best_slope <- (s_y/s_x) * best_slope #scale
-    best_inter <- s_y * best_inter
-    
+
+    orig_slope <- (cov(x, y)/var(x))
+    orig_inter <- mean(y) - orig_slope*mean(x)
+
+    best_slope <- (s_y/s_x) * orig_slope #scale
+    best_inter <- s_y * orig_inter
+
     if (s_x != 0){
-      paste0("Best Slope: ", round(best_slope, 3), "\nBest Intercept: ", round(best_inter, 3))
+      paste0("Original Slope: ", round(orig_slope, 3),
+             "\nOriginal Intercept: ", round(orig_inter, 3),
+             "\nCurrent Slope: ", round(best_slope, 3),
+             "\nCurrent Intercept: ", round(best_inter, 3))
     } else {
-      paste0("Best Slope: ", round(best_slope, 3), "\nBest Intercept: ", round(best_inter, 3), "\n\nRemember that, when X = 0, the best linear predictor of Y is simply its (potentially rescaled) mean!")
+      paste0("Original Slope: ", round(orig_slope, 3),
+             "\nOriginal Intercept: ", round(orig_inter, 3),
+             "\nCurrent Slope: ", round(best_slope, 3),
+             "\nCurrent Intercept: ", round(best_inter, 3),
+             "\n\nRemember that, when X = 0, the best linear predictor of Y is simply its (potentially rescaled) mean!")
     }
-    
-    
   })
-  
+
   output$regPlot_rescale <- renderPlot({
-    
+
     set.seed(19)
-    
+
     x <- rnorm(50, 2, 5)
```

Comment:
you seem to have a different variance here. why?
* why do you create the data twice?
* could you just reduce the error variance (instead of 10 use 1) and the estimates would come closer to the DGP? it's kidn of annoying that we see 8.1 instead of 10.0 for the original estimates.